lib/install.ps1

```diff
@@ -529,19 +528,15 @@ function dl_urls($app, $version, $manifest, $bucket, $architecture, $dir, $use_c
 
         # work out extraction method, if applicable
         $extract_fn = $null
-        if($fname -match '\.zip$') {
+        if ($manifest.innosetup) {
+            $extract_fn = 'extract_inno'
+        } elseif($fname -match '\.zip$') {
             $extract_fn = 'extract_zip'
         } elseif($fname -match '\.msi$') {
             # check manifest doesn't use deprecated install method
             $msi = msi $manifest $architecture
```

Comment:
This variable is not needed. It could be just inside if

```suggestion
if (msi $manifest $architecture) {
    warn deprecated
} else {
    $extract_fn
}
```